develop/reference_sql.rst

```diff
@@ -70,6 +70,132 @@ As an example, if you want to run the hll_union aggregate function on your data
 
 You can then call sum(hll_column) to roll up those columns within the database. Please note that these custom aggregates need to be created both on the coordinator and the workers.
 
+.. _topn:
+
+Estimating Top N Items
+~~~~~~~~~~~~~~~~~~~~~~
+
+Similar to how HLL efficiently estimates the number of distinct items in a set, the `TopN PostgreSQL extension <https://github.com/citusdata/postgresql-topn>`_ simultaneously counts many items. It, too, stores an approximation in a bounded amount of space. It is also pre-installed on Citus Cloud.
```

Comment:
Could you provide a better description of TopN. The current one just says: "the `TopN PostgreSQL extension <https://github.com/citusdata/postgresql-topn>`_ simultaneously counts many items.". 

Maybe describe that it is an approximation algorithm, and provide motivation on why it might be useful:
 For example, the section here: https://github.com/citusdata/postgresql-topn#why-use-topn has some good information on why top-N.